sdk/search/search-documents/CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 # Release History
 
+## 11.3.0 (Unreleased)
+
+### Features Added
+
+- Enabled national cloud support for Azure Search SDK. Please refer [#22887](https://github.com/Azure/azure-sdk-for-js/pull/22887) for further details.
+- Support for TokenCredential has been added. With this addition, the Search SDK supports authentication via AAD.
+
+### Bugs Fixed
+
+- Converted the complex fields correctly within the Search Fields. Please refer [#16489](https://github.com/Azure/azure-sdk-for-js/issues/16489) for more details.
+- Fixed the typos `anayzerName` to `analyzerName` in `convertFieldsToPublic` method of `serviceUtils.ts`.
+- Fixed the issue with the presence of recursive structure while uploading documents. Please refer [#15656](https://github.com/Azure/azure-sdk-for-js/issues/15656) for further details.
+
```

Comment:
We want to keep our changelog up-to-date with all the released packages information. So can we add all the preview packages information in the changelog from 11.3.0-beta.1 to 11.3.0-beta.7? and add a note like [this](https://github.com/Azure/azure-sdk-for-net/blob/Search_11.4.0_GA/sdk/search/Azure.Search.Documents/CHANGELOG.md#note)?